### ISSUE TITLE
[BUG] - SHFQC: Number of measurements executed do not match defined

### ISSUE THREAD

**Issue by GlaserN** [zhinst/laboneq#14]
**Describe the bug**
Running two experiments directly after each other gives with high probability the following error:

```
The number of measurements (90) executed for device dev12190 on channel 0 does not match the number of measurements defined (26112). Probably the time between measurements or within a loop is too short. Please contact Zurich Instruments.
```
After a retry the reported results do not look consistent with the expected results.
**To Reproduce**
Run the following experiment twice or multiple times in the same session on the SHFQC (4 channel):

[SHFQC_measurement.zip](https://github.com/zhinst/laboneq/files/10178288/SHFQC_measurement.zip)

**Expected behavior**
Measurements should be fetched correctly.

**Versions used:**
 - laboneq: 1.4
 - zhinst-core
 - others

**Context**
- [ ] Do you know a workaround for the issue? If yes, please provide a short description. 
- [ ] Does the issue block your experiments?

**Comment by mforoozandeh** [comment]
Hi @GlaserN,
Thanks for reporting this. We have started looking into this issue and will update you as soon as we have a fix.

**Comment by QuantumClemens** [comment]
Hi @GlaserN - This will be fixed in the upcoming 1.6 release, which should come early next week - there has been some delays due to bugs discovered in testing the release
